services/supervisor/tests/supervisor/main.rs

```diff
@@ -1024,7 +1043,11 @@ fn test_id_assignment_sparse() {
     // Create testkit with builtin instance with ID 100.
     let mut testkit = TestKitBuilder::validator()
         .with_logger()
-        .with_default_rust_service(DecentralizedSupervisor::new())
+        .with_rust_service(Supervisor)
```

Comment:
These 4 lines look like an fault of `TestKitBuilder` interface, but I'm not sure how it could be improved. Declaring methods for creating `InstanceInitParams` for a simple and decentralized supervisor - i.e., using something like `.with_instance(Supervisor::decentralized())` - could shorten things a bit.